portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -31,6 +34,15 @@ public long getId() {
         return id;
     }
 
+    /**
+     * Returns the comment's nickname.
+     * 
+     * @return The comment's nickname.
+     */
```

Comment:
JavaDoc comments are actually optional on trivial getters and setters. I recommend removing them.

Reply:
Yeah I might just end up removing the get functions altogether since I never use them and don't plan to use them.

Reply:
Removed the get functions and resolving this.